### DISCUSSION TITLE
OpenId installation error on pimcore 10.6.9v

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [pimcore/pimcore#17328] [Q&A / Support]
### Pimcore version

10.6.9

### Steps to reproduce

Hi ,

I have tried to install openId for my project in the pimcore 10.6.9 version. Im getting below error attached
i ran the command **composer require pimcore/openid-connect**

can anyone help me to find out the cause and solve this issue?

![openid](https://github.com/pimcore/pimcore/assets/78895325/9fb33481-bd94-43e0-8929-1d344642aa25)

### Actual Behavior

openId not installing

### Expected Behavior

Should work without issue

**Comment by kingjia90** [comment]
OpenID connect bundle is an [enteprise](https://pimcore.com/docs/platform/OpenID_Connect/) bundle, you won't be able to directly install it from the public repository, you need to set up the private repository (auth token, repo url) according to the instructions you may have received as an Enterprise Customer. 

If you already have set it up, then please check if the required version is `^1` or so.
